deepmd/utils/data.py

```diff
@@ -249,6 +254,25 @@ def check_test_size(self, test_size):
         else:
             return None
 
+    def get_item(self, index: int) -> dict:
```

Comment:
It seems this method can only be used by pt (since it calls `reformat_data_torch`).